.github/workflows/upgrade_downgrade_test_backups_e2e.yml

```diff
@@ -92,12 +92,10 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y mysql-server mysql-client make unzip g++ etcd-client etcd-server curl git wget eatmydata
         
-        # Running on CNCF ARC Runners no longer needs this
-        # Leaving this in as a tombstone for now
-        #sudo service mysql stop
-        #sudo service etcd stop
-        #sudo ln -s /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/disable/
-        #sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
+        sudo service mysql stop
+        sudo service etcd stop
+        sudo ln -s /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/disable/
+        sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
```

Comment:
We're seeing failures in multiple PRs on this job that very suspiciously look like the errors we get on the GitHub runners when this is missing. The problem is that it's not always 100% failing, but we fail still regularly and it looks like `mysqlctld` can't start MySQL which can be due to these kinds of app armor rules still being enforced. 

This also matches where I can't reproduce the problem locally. 

Also FYI @jeefy @krook & @RobertKielty